content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,71 @@
+---
+permalink: "rackspace-fabric-faq"
+audit_date: '2022-03-17'
+type: article
+title: "Rackspace Fabric FAQ"
+created_date: '2022-03-17'
+created_by: Asmita Nakwa
+last_modified_date: '2022-03-17'
+last_modified_by: Asmita
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+### General
+{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}
+
+If this is cloud account admin, they can review the following link (https://account.rackspace.com/event-history). It should help them understand any recent changes/access to their account.
+{{</accordion>}}
+
+{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
+
+The customer needs to add the firewall as a device and the **Add VPN User** template will be visible in common requests.
+{{</accordion>}}
+
+{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}
+
+MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team.
+{{</accordion>}}
+
+{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
+
+The customer will get notification only if the device in question is opted in.{{</accordion>}}
+
+{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}
+
+There is no primary contact on the account. That is required for the account to work. I believe it can be set from Core.
+{{</accordion>}}
+
+### Permission
+
+{{<accordion title="I am unable to upload documents and tried multiple times on multiple tickets." col="in" href="accordion1">}}
+
+Opt in to the ticketing beta using the ticketing drop down in the global navigation.
+{{</accordion>}}
+
+{{<accordion title="I have raised ticket using Rackspace BETA version of ticketing system, but unable to edit the ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}
+
+User need to have either Edit or Admin permission on the device that is attached to a ticket in order to update the ticket.
```

Comment:
So again, we want to be a little more general and not mention the actual permission names for the security reasons and these docs are public.
If a device is on a ticket, and they don't have permissions to that device, then for sure that is one possible cause.
Another cause is they just might not have Ticketing permissions (specifically the Admin permission). They would might be an observer. An observer can only view tickets and can't update a ticket.
So you would have to figure out how to safely word this.

Reply:
Either way, they need to contact an Admin from the customers side.
If thats not possible, then they need to contact their Customer Success Manager or support.

Reply:
Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information)  and select appropriate account and product. 
Need to contact admin (customer side) or need to contact their Customer Success Manager or Support Team.


Please confirm this.